src/commands/Weather.ts

```diff
@@ -0,0 +1,482 @@
+        const metarEmbed = new MessageEmbed()
+            .setTitle(title)
+            .setColor("#0099ff")
+            .setTimestamp();
+        try {
+            const {raw, readable} = await this._avwxManager.getMetar(icao);
+            if (rawData) {
+                metarEmbed.setDescription(raw);
+            } else {
+                metarEmbed
+                    .addFields(
+                        {
+                            name: "Raw Report",
+                            value: raw
+                        },
+                        {
+                            name: "Readable Report",
+                            value: readable
+                        }
+                    );
+            }
+        } catch (err) {
+            logger.error(`[${client.shard.ids}] ${err}`);
+            metarEmbed.setColor("#ff0000").setDescription(`${interaction.member}, ${err.message}`);
+        }
+        metarEmbed.setFooter({
+            text: `${client.user.username} • This is not a source for official briefing • Please use the appropriate forums • Source: AVWX`
+        });
+        return InteractionUtils.replyOrFollowUp(interaction, {
+            embeds: [metarEmbed]
+        });
+    }
+
+    @Slash("atis", {
+        description: "Gives you the live ATIS for the chosen airport"
+    })
+    @Guard(NotBot, RequiredBotPerms({
+        textChannel: ["EMBED_LINKS"]
+    }))
+    private async atis(
+        @SlashOption("icao", {
+            autocomplete: (interaction: AutocompleteInteraction) => InteractionUtils.search(interaction, AvwxManager),
+            description: "What ICAO would you like the bot to give ATIS for?",
+            type: "STRING",
+            required: true
+        })
+            icao: string,
+        interaction: CommandInteraction,
+        client: Client
+    ): Promise<void> {
+        const atisEmbed = new MessageEmbed()
+            .setTitle(`ATIS for ${icao.toUpperCase()}`)
+            .setFooter({
+                text: `${client.user.username} • This is not a source for official briefing • Please use the appropriate forums • Source: AVWX`
+            })
+            .setTimestamp();
+        try {
+            const {speech} = await this._avwxManager.getMetar(icao);
+            atisEmbed.setDescription(speech);
+        } catch (error) {
+            logger.error(`[${client.shard.ids}] ${error}`);
+            atisEmbed.setColor("#ff0000").setDescription(`${interaction.member}, ${error.message}`);
+        }
+
+        return InteractionUtils.replyOrFollowUp(interaction, {
+            embeds: [atisEmbed]
+        });
+    }
+
+
+    // TODO: this needs looked, it works, but might need improvements
+    @Slash("atis-voice", {
+        description: "Gives you the live ATIS as voice for the chosen airport"
+    })
+    @Guard(NotBot, RequiredBotPerms({
+        textChannel: ["EMBED_LINKS", "ADD_REACTIONS"],
```

Comment:
I think we can remove the reaction permissions since now the buttons are used.
```suggestion
        textChannel: ["EMBED_LINKS"],
```